gateway/src/main/java/io/zeebe/gateway/impl/job/ActivateJobsHandler.java

```diff
@@ -97,14 +105,19 @@ private void activateJobs(
           response -> false);
     } else {
       // enough jobs activated or no more partitions left to check
-      jobTypeToNextPartitionId.put(jobType, partitionIdIterator.getCurrentPartitionId());
-      responseObserver.onCompleted();
+      try {
+        responseObserver.onCompleted();
+      } catch (Exception e) {
+        // Cannot close the stream. Nothing to do
```

Comment:
Are there any possible downsides to suppressing the exception? Could we miss out on useful information?

Reply:
I guess no problem. Activated jobs are already sent to the client. If there was any problem there it will be logged. This is called only to close the stream. I don't think it is much useful to log failure here.